docs/using-mycroft-ai/basic-commands.md

```diff
@@ -1,104 +1,328 @@
+###
+### Spelling - *Mycroft will spell any word that is understood by the Speech-To-Text system.*
+> Hey Mycroft, spell **aardvark** <BR>
+> Hey Mycroft, what is the spelling of **aardvark**? <BR>
+> Hey Mycroft, how do you spell word **aardvark**?
+###
+### Stock - *This Skill has been disabled.*
+###
+### Stop - *Tell Mycroft to STOP doing whatever it is doing.  Plus miscellaneous Device hardware commands.*
+##### Stop - *This command should terminate any command that is in progress*
+> Hey Mycroft, stop <BR>
+> Hey Mycroft, silence <BR>
+> Hey Mycroft, shut up
+##### Reboot the Mycroft Device
+> Hey Mycroft, reboot
+##### Shutdown the Mycroft Device
+> Hey Mycroft, shutdown <br>
+> Hey Mycroft, power down
+##### Activate WIFI 
+> Hey Mycroft, setup wifi <br>
+> Hey Mycroft, configure wifi
+##### SSH Access 
+> Hey Mycroft, enable/disable SSH login <br>
+> Hey Mycroft, allow/deny remote connection
+###
+### Support Helper - *Captures troubleshooting information from the Mycroft Device.*
+> Hey Mycroft, contact support <br>
+> Hey Mycroft, create a support ticket <br>
+> Hey Mycroft, troubleshoot my device <br>
+> Hey Mycroft, create a troubleshoot request <br>
+
+*Captured information is useful for your own debugging and for communicating with Support. The information is emailed to the address registered at home.mycroft.ai.*
+###
+### Timer - *Manages a collection of timers.*
+*Timers are requested with durations expressed in hours, minutes, and/or seconds.  Note: If you wish to command a specific expiration time (e.g. 'at 3:05pm'), please use either an Alarm or a Reminder.  A timer may optionally be given a name.  The name is useful in queries, subsequent commands, and for display purposes.  All timers beep.*
+##### Start a timer - *User provides a timer name and a duration*
+> Hey Mycroft, I need a timer named **turkey** for **15 minutes**<br>
+> Hey Mycroft, start a **10-minute** timer called **time to go**
+##### Start a timer - *User provides a timer name, Mycroft will ask for the duration.*
+> Hey Mycroft, set a timer to **take out the trash**
+##### Start a timer - *Mycroft will ask for the duration.  No name is associated with this timer.*
+> Hey Mycroft, Start a timer<br>
+> Hey Mycroft, Begin timer
+##### Start a timer - *User provides just the duration.  No name is associated with this timer.*
+> Hey Mycroft, ping me in 2 hours
+##### Query a timer
+> Hey Mycroft, how's my **turkey** timer? <br>
+> Hey Mycroft, what's the status on all timers? <br>
+> Hey Mycroft, how much time is left?
+##### Cancel the timer
+> Hey Mycroft, cancel the timer <br>
+> Hey Mycroft, kill the **turkey** timer
+##### Cancel all timer
+> Hey Mycroft, cancel all timers <br>
+> Hey Mycroft, delete all timers
+
+*Note: Timers have durations less than 24-hours.  When a Timer is requested with a duration greater than 24-hours, an Alarm is set instead.*<br>
+*Note: Mycroft refers to a timer by it's original duration along with the optional User supplied name.*
```

Comment:
you can also refer to them by their ordinal, eg
> set a 5 minute timer
> set a 10 minute timer
> cancel the 2nd timer

Reply:
Right.